### ISSUE TITLE
Move the "Preview" text somewhere less intrusive

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/markdown-editor#345]
![image](https://user-images.githubusercontent.com/40569793/79174067-8a459000-7dc7-11ea-900c-e6b93f8eaba1.png)

I think we should move the "Preview" text somewhere less intrusive. When working with small amounts of markdown text, the "preview" label gets in the way and is distracting. I'm not even sure what the "preview" label is communicating. If the label is referring to the editor on the right side of the screen, the label should be above that editor, and there should be a label above the left editor saying "markdown" or whatever is appropriate and accurate.

**Comment by arteevraina** [comment]
I would like to work on this.

**Comment by Michael-Grover** [comment]
@arteevraina sure, thanks
